lib/landable.rb

```diff
@@ -10,6 +10,7 @@
 require "landable/core_ext/ipaddr"
 
 require "lookup_by"
+require "figgy"
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.